### PR TITLE
Feature Config for env

### PR DESCRIPTION
**Pull request by SyedSibtainRazvi**
# Etiquettes of raising a PR

PR reviews are a way to request peers to give feedback on the code that is going
to the mainline. This means, that PR should be in a reviewable state. Please ensure
that the following checkboxes are ticked. If any of them are not ticked, then
the PR is not ready for review

- [x] Did you test changes locally?
- [x] Did you check PR contents after opening the PR

# Related issue

- [Integrate Config for env](https://github.com/code-mancers/strapi-plugin-stellate/issues/7)

# Functionalities added or changed

- Added configuration for config file

# Additional info
# Checklist

- [ ] Added/Updated Story
- [ ] Tests
- [ ] Translations
- [ ] Documentation
- [ ] Uploaded screenshots
- [x] Verified on preview URL once

# Screenshots

## Page(s) if applicable

|   Device    |            Design             |           Implementation            |
| :---------: | :---------------------------: | :---------------------------------: |
| **Desktop** | ** Figma design screenshot ** ||

### CODE REVIEW

File: server/helpers/pluginConfig.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const pluginId = require("./pluginId");
+
+const getPluginConfig = (strapi) => {
```
Comment (SujayPrabhu96) [inline server/helpers/pluginConfig.js]:
```suggestion
const getPluginConfig = (strapi) => strapi.plugin(pluginId).config;
```

File: server/services/cache-service.js [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
 module.exports = ({ strapi }) => ({
   async cacheService() {
     try {
+      const config = buildConfig(strapi);
+      if (!config) {
+        throw "Missing configuration";
+      }
       const data = await axios({
-        url: process.env.STELLATE_URL,
+        url: `${config.stellateUrl}`,
```
Comment (SujayPrabhu96) [inline server/services/cache-service.js]:
```suggestion
        url: config.stellateUrl,
```

File: server/services/cache-service.js [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
         method: "POST",
         headers: {
           "Content-Type": "application/json",
-          "stellate-token": process.env.STELLATE_TOKEN,
+          "stellate-token": `${config.stellateToken}`,
```
Comment (SujayPrabhu96) [inline server/services/cache-service.js]:
```suggestion
          "stellate-token": config.stellateToken,
```

File: server/services/cache-service.js [hunk 4]
```diff
@@ -25,12 +29,16 @@ module.exports = ({ strapi }) => ({
   },
   async refreshCollectionCache(query) {
     try {
+      const config = buildConfig(strapi);
+      if (!config) {
+        throw "Missing configuration";
+      }
       const data = await axios({
-        url: process.env.STELLATE_URL,
+        url: `${config.stellateUrl}`,
```
Comment (SujayPrabhu96) [inline server/services/cache-service.js]:
```suggestion
        url: config.stellateUrl,
```

File: server/services/cache-service.js [hunk 5]
```diff
@@ -25,12 +29,16 @@ module.exports = ({ strapi }) => ({
...
         method: "Post",
         headers: {
           "Content-Type": "application/json",
-          "stellate-token": process.env.STELLATE_TOKEN,
+          "stellate-token": `${config.stellateToken}`,
```
Comment (SujayPrabhu96) [inline server/services/cache-service.js]:
```suggestion
          "stellate-token": config.stellateToken,
```

**Review by SujayPrabhu96 — CHANGES_REQUESTED** [review]

File: server/helpers/pluginConfig.js (hunk 1 above)
Comment (SyedSibtainRazvi) [reply server/helpers/pluginConfig.js]: Updated

File: server/services/cache-service.js (hunk 2 above)
Comment (SyedSibtainRazvi) [reply server/services/cache-service.js]: Updated

File: server/services/cache-service.js (hunk 5 above)
Comment (SyedSibtainRazvi) [reply server/services/cache-service.js]: Updated all

**Review by SujayPrabhu96 — APPROVED** [review]

**Pull request merged**